src/yunohost/app.py

```diff
@@ -156,10 +156,18 @@ def app_info(app, full=False):
 
 
 def _app_upgradable(app_infos):
+    from packaging import version
```

Comment:
Maybe we should add "python-packaging" in debian/control ?

Reply:
I think it's not mandatory because the package `python-pkg-resources` also contains this module. 
`yunohost` depends of ` python-jinja2` and ` python-jinja2` depends of `python-pkg-resources` so it should be installed on all yunohost instances.

```
root@ynh-dev1:~# apt-cache rdepends --installed python-pkg-resources
python-pkg-resources
Reverse Depends:
  python-setuptools
  python-jinja2
  python-chardet
root@ynh-dev1:~# apt-cache rdepends --installed python-jinja2
python-jinja2
Reverse Depends:
  yunohost
  yunohost
```